repos/system_upgrade/el7toel8/actors/cupsmigrate/actor.py

```diff
@@ -0,0 +1,339 @@
+from leapp.actors import Actor
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import Report, Cupsmigrationmodel
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+import os
+from re import compile, findall, search, sub, MULTILINE
+
+"""
+CUPS configuration files
+"""
+CUPSD_CONF = '/etc/cups/cupsd.conf'
+CUPSFILES_CONF = '/etc/cups/cups-files.conf'
+
+"""
+Names for better navigation through change list
+"""
+
+INCLUDE = 0
+DIGEST = 1
+ENV = 2
+PRINTCAP = 3
+
+CERTKEY = 0
+
+KEY = 0
+NAMES = 1
+NEWNAME = 2
+SOURCE = 3
+TARGET = 4
+
+"""
+List of changed directives
+
+Scheme:
+Dictionary key # Directives # Input configuration file # Output configuration file
+
+Some changes happened only in one configuration file, so the last member of list is 'None'.
+"""
+cupsd_changes_list = [
```

Comment:
cupsd_changes_list could be a Python's dictionary (or even namedtuples) what would simplify this code a bit. Instead of:
```python
INCLUDE = 0
(...)
cupsd_changes_list [['include', (...)]]
(...)
cupsd_changes_list[INCLUDE]
```
we can have:

```python
cupsd_changeslist {'include': ['include', (...)]}
cupsd_changes['include']
```
Also, since this is used at a single place afaict, moving it closer to where it is used (inside function) can help readability in my opinion.

Reply:
I chose different approach in the end.